### DISCUSSION TITLE
Publicly accessible apps for non-authenticated users

### DISCUSSION THREAD

**Discussion by lsoica** [Budibase/budibase#4805] [Help]
Is it possible to configure an app to be publicly accessible for anonymous users ?

What I've tried: 

- Setting public access level for data sources
- Setting public access level for screens

If I open the app in a browser where I am not authenticated, I get:
```
{"message":"Database does not exist.","status":404}
```

I would have expected at least a redirect to the login page. Is this expected ?

**Comment by mjashanks** [comment]
It should work ok - can you browse directly to one of the public screens e.g. `https://your-domain.com/app/Your-App/#public-screen` ?

**Reply by lsoica** [reply]
Same behaviour. {"message":"Database does not exist.","status":404}

**Reply by HideoKuze** [reply]
did you manage to fix this?

**Reply by lsoica** [reply]
No. Same behavior with latest version.

**Reply by Soufas** [reply]
It worked for me with this URL: https://your-domain.com/app/Your-App#/public-screen . Note that the Hashtag is before the slash not after.

**Reply by lsoica** [reply]
Tried this aswell. Nothing worked for me. But I'm on k8s, that could be a reason.
